lib/middleware.js

```diff
@@ -64,43 +61,96 @@ Middleware.prototype.findMatchingRule = function(message) {
 
 Middleware.prototype.parseMetadata = function(message) {
   var result = {
-    domain: this.slackClient.getTeamDomain(),
-    channel: this.slackClient.getChannelName(message.item.channel),
-    timestamp: message.item.message.ts
+    channel: this.slackClient.getChannelName(message.channel),
+    timestamp: message.message.ts,
+    url: message.message.permalink
   };
   result.date = new Date(result.timestamp * 1000);
   result.title = 'Update from #' + result.channel +
     ' at ' + result.date.toUTCString();
-  result.url = 'https://' + result.domain + '.slack.com/archives/' +
-    result.channel + '/p' + result.timestamp.replace('.', '');
   return result;
 };
 
-function fileGitHubIssue(that, msgId, message, githubRepository) {
-  var metadata = that.parseMetadata(message);
+function getReactions(that, msgId, message) {
+  var domain = that.slackClient.getTeamDomain(),
+      channelName = that.slackClient.getChannelName(message.item.channel),
+      timestamp = message.item.ts,
```

Comment:
Nope. This is based on the actual JSON structure I observed after launching it for real, which was different from what the API docs originally seemed to indicate. (I filed a doc change request with Slack; they responded that they're looking into it.)